### PR TITLE
ResNet18||CIFAR10 Inference time|| Baidu-USA-GAIT

### PR DESCRIPTION
**Pull request by BAIDU-USA-GAIT-LEOPARD**
we update the Json file, please only review the latest pull request.

### CODE REVIEW

File: CIFAR10/inference/BAIDU-USA-GAIT_ResNet18_1GTX1080_pytorch.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+	    "version": "v1.0",
+	    "author": Baidu USA GAIT LEOPARD team: Baopu Li, Jiazhuo Wang, Zhiyu Cheng, Haofeng Kou, Yingze Bao
```
Comment (codyaustun) [inline CIFAR10/inference/BAIDU-USA-GAIT_ResNet18_1GTX1080_pytorch.json]: This is not valid JSON. Please fix and make sure the unit tests pass. From the root directory of this repo, you can run the unittests locally with the following:

```
pip install -r requirements.txt
pytest
```

**Review by codyaustun — CHANGES_REQUESTED** [review]
@BAIDU-USA-GAIT-LEOPARD I'll finish reviewing this PR once the file extension is fixed and the unit tests pass.

**Comment by codyaustun** [comment]
@BAIDU-USA-GAIT-LEOPARD it looks like you deleted the JSON file. Should I close this PR and only review #77 ?

**Comment by BAIDU-USA-GAIT-LEOPARD** [comment]
> @BAIDU-USA-GAIT-LEOPARD it looks like you deleted the JSON file. Should I close this PR and only review #77 ?

Yes please and thanks.

**Pull request closed without merging**